lib/userInterface.js

```diff
@@ -0,0 +1,59 @@
+const readline = require("readline");
+const chalk = require("chalk");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+const BowlingScoreCard = require("./bowlingScoreCard");
+
+class UserInterface {
+  constructor() {
+    this.scoreCard = new BowlingScoreCard();
+  }
+
+  start() {
+    console.log(chalk.cyan("Welcome to the Bowling Score Card Application!"));
+    this.playFrame(1, 1);
+  }
+
+  playFrame(frameNum, rollNum) {
+    console.log(chalk.yellow(`Frame ${frameNum} - Roll ${rollNum}`));
+    rl.question("Enter the number of pins down: ", (pinsDown) => {
+      const roll = parseInt(pinsDown);
```

Comment:
This method could definitely do with being divided into several helper methods, but you are probably aware of this and I understand that your hectic weekend probably didn't allow much time for refactoring.

Reply:
100% I didn't spend a great deal of time on refactoring, especially with the UI class. Most of my time was spent trying to learn about readline.